VISION.md

```diff
@@ -0,0 +1,14 @@
+# Material UI’s Vision
+
+Our vision is to provide an elegant implementation of the [Material Design guidelines](https://material.io/guidelines/) and more.
+
+The Material Design guidelines are an incredible starting point, but they do not provide guidance on all aspects or needs of an application. In addition to the guidelines-specific implementation, we want Material-UI to become whatever is generally useful for application development, all in the spirit of the Material Design guidelines.
+
+Therefore, Material-UI will be not only be an implementation of the Material Design guidelines, but a general use UI library of components that are needed by many. This generalized use doesn't imply any other design methodology. It also means we will have components or combinations that are simply not addressed in the design guidelines.
+
+We will focus on providing all the low-level tools needed to build a rich user-interface with React. Once we implement the Material design specification (which is a bar set quite high), you should be able to take advantage of it for you own business with any style customization needed. We want to see companies succeeding using Material-UI in a way that matches their brand. We don't want them to feel that their UI simply looks like another Google product.
+
+From a user point of view, we want Material-UI
+- to deliver on fully encapsulated/composable react components
+- to be themable/customizable
+- to promote developer joy
```

Comment:
@oliviertassinari I think it's great!

Here are some other bullets that I'd like to suggest.

- to be cross browser compatible and accessible

- to promote developer joy, a sense of community, and an environment where new and experienced developers can learn from each other